### ISSUE TITLE
Update examples and code to do conditional displayio refactoring imports

### ISSUE THREAD

**Issue by dhalbert** [adafruit/Adafruit_CircuitPython_ILI9341#34]
e.g. like:
```py
try:
    from busdisplay import BusDisplay
except ImportError:
    from displayio import Display as BusDisplay
```
and include `fourwire` as well above.

**Comment by dhalbert** [comment]
See https://github.com/adafruit/Adafruit_CircuitPython_SSD1331/pull/18/files for an example

**Comment by tekktrik** [comment]
Resolved via #35
